src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Organization/Position/Position.razor

```diff
@@ -9,7 +9,7 @@
     <AutoHeight>
         <HeaderContent>
             <div class="d-flex mb-6">
-                <SearchTextField @bind-Value="Search" />
+                <SSearch @bind-Value="Search" Style="max-width: 340px;" />
```

Comment:
add Paramter Width

Reply:
max-wdith packaging to MASA.Stack.Component